File: Jarvis.Web/MG.Jarvis.Web/ClientApp/src/app/extranet/viewmodel/policiesviewmodel/cancellationpolicyclausesviewmodel.ts
import { IObjectWithState } from '../../../common/iobjectwithstate';
import { ObjectState } from '../../../common/enums';

export class CancellationPolicyClausesViewModel implements IObjectWithState {
    cancellationPolicyClausesId: number;
    daysBeforeArrival: number;
    percentageCharge: number;
    cancellationChargesId: number;
    cancellationPolicyId: number;
    objectState: ObjectState;
}

File: Jarvis.Web/MG.Jarvis.Api.Extranet/ViewModel/CancellationPolicyClausesViewModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MG.Jarvis.Api.Extranet.ViewModel
{
    public class CancellationPolicyClausesViewModel : BaseViewModel
    {
        public int CancellationPolicyClausesId { get; set; }
        public int DaysBeforeArrival { get; set; }
        public decimal PercentageCharge { get; set; }
        public int CancellationChargesId { get; set; }
        public int CancellationPolicyId { get; set; }
    }
}
